### DISCUSSION TITLE
Is age a trusted extension?

### DISCUSSION THREAD

**Discussion by ShravanSunder** [apache/age#1631] [General] [answered]
I was wondering if age is a trusted extension and can be installed in supabase postgres databases
https://www.postgresql.org/about/featurematrix/detail/347/

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL_trusted_language_extension.html

**Comment by jrgemignani** [comment, marked as answer]
@ShravanSunder For the first link, yes. However, for the second link, no. AGE is not a Trusted Language Extension as it is written in C.

**Comment by aked21** [comment]
@ShravanSunder .. It seems there is a bit of talk happening at supabase to support Apache AGE. 
[https://github.com/orgs/supabase/discussions/13263](url)

**Reply by markgomer** [reply]
This link seems broken, try this one [https://github.com/orgs/supabase/discussions/13263](https://github.com/orgs/supabase/discussions/13263)

There's an open discussion there to vote for the next extensions to be added on Supabase:
[https://github.com/orgs/supabase/discussions/679#discussioncomment-7323662](https://github.com/orgs/supabase/discussions/679#discussioncomment-7323662)

**Reply by mingfang** [reply]
While I love AGE but I don't think it's ready for prime time yet.
There are critical bugs with some of the basic features that would make a very bad first impression to a wider audience.
I suggest we wait until those are fixed before promoting it.

**Reply by mingfang** [reply]
I'm glad to report that recent work from @jrgemignani has resolve all of the critical issues I had.

**Reply by ShravanSunder** [reply]
amazing!  is it possible to reach out to supabase and advocate for them to add the extension?

i added a comment here:  https://github.com/orgs/supabase/discussions/679#discussioncomment-9201912
